docs/advanced-usage/using-proxies.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Proxies
+description: Learn how to proxy the Clerk Frontend API through your domain.
+---
+
+# Proxying the Clerk Frontend API
+
+Clerk supports two methods for configuring the Clerk Frontend API: CNAME and Proxy. 
+
+CNAME is the simplest way to configure the Frontend API but, if you're unable to use the CNAME approach, or would like more control over your integration with Clerk, you can use a proxy.
```

Comment:
:upside_down_face:  "Setting up CNAME records and using DNS is the simplest way"???

Maybe split the sentences into shorter ones? Sorry if I'm being nit-picky...